gammapy/background/off_data_background_maker.py

```diff
@@ -230,7 +230,46 @@ def save_models(self, modeltype):
             Type of the background modelisation
         """
         for ngroup in range(self.ntot_group):
-            self.save_model(modeltype, ngroup)
+            self.save_model(modeltype, ngroup, smooth)
+
+    def smooth_models(self, modeltype):
+        """Smooth the bkg model for each group
+
+        Parameters
+        ----------
+        modeltype : {'3D', '2D'}
+            Type of the background modelisation
+
+        Returns
```

Comment:
This doesn't return anything (and shouldn't, right?).
So remove this from the docstring?